backend/SlothOrganizer/SlothOrganizer.Web.Tests.Integration/Setup/DtoProvider.cs

```diff
@@ -55,7 +55,7 @@ public static NewDashboardDto GetNewDashboard()
             };
         }
 
-        public static NewTaskDto GetNewTask()
+        public static NewTaskDto GetNewTask(TaskRepeatingPeriod period = TaskRepeatingPeriod.None)
```

Comment:
I guess there is some sense to separate responsibility - split providers, inject required providers into specific tests.
Also, you may have not only one test base - specific test bases for tests. In that way you won't need to have such complex setup for every tests. Setup only required data for test.

Reply:
Split dto providers into auth-related and dashboard-related, separated AuthorizedTestBase and TaskTestBase